search-service/src/main/kotlin/com/egm/stellio/search/util/QueryUtils.kt

```diff
@@ -14,20 +14,24 @@ import java.util.Optional
 fun parseAndCheckQueryParams(
     pagination: ApplicationProperties.Pagination,
     requestParams: MultiValueMap<String, String>,
-    contextLink: String
+    contextLink: String,
+    pointOfTimeIsNeeded: Boolean = false
 ): TemporalEntitiesQuery {
+    val queryParams = parseAndCheckParams(
+        Pair(pagination.limitDefault, pagination.limitMax),
+        requestParams,
+        contextLink
+    )
+    if (queryParams.types.isEmpty() && queryParams.attrs.isEmpty() && pointOfTimeIsNeeded)
+        throw BadRequestDataException("Either type or attrs need to be present in request parameters")
```

Comment:
I don't see what `pointOfTimeIsNeeded` is doing here.

Reply:
beacuse it's required only when we do a get entities and not for a get by id

Reply:
ok, so a renaming must be done, to make clear what this last condition is about.

Reply:
Ok I rename attribute

Reply:
What is the best `temporalQueryRequired` or `queryEntitiesAsked`

Reply:
`inQueryEntities`